include/SampleThumbnail.h

```diff
@@ -105,6 +103,7 @@ class LMMS_EXPORT SampleThumbnail
 
 		Thumbnail zoomOut(float factor) const;
 
+		Peak* data() { return m_peaks.data(); }
```

Comment:
I think I added this, but can you make it return `const Peak*` for me instead?

Reply:
Wait nevermind I see there was an issue somewhere. Let me review a bit more.

Reply:
I think the dereferencing logic was solid, it's odd how it appeared. I think it must be some problem with the bounds. I got this crash too I believe so I'll confirm if the fix works or not.